gnome/gnome-radio/Portfile

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gnome-radio
+version             0.1.2
+set branch          [join [lrange [split $version .] 0 1] .]
+
+categories          gnome
+platforms           darwin
+license             GPL-3+
+maintainers         {gnome.org:ole @oleaamot}
```

Comment:
Can `openmaintainer` be added?

Reply:
When I asked for it to be on one line, I did not mean to remove `openmaintainer`.

Reply:
Fixed.